L1Trigger/Configuration/python/SimL1Emulator_cff.py

```diff
@@ -62,7 +62,8 @@
 _phase2_siml1emulator += hgcalTriggerPrimitives
 
 from Configuration.Eras.Modifier_phase2_hgcal_cff import phase2_hgcal
-phase2_hgcal.toReplaceWith( SimL1Emulator , _phase2_siml1emulator )
+from Configuration.Eras.Modifier_phase2_hgcalV9_cff import phase2_hgcalV9
```

Comment:
this change looks unrelated to the rest of the PR, it is just #24418 already merged picked up